### ISSUE TITLE
Wifi setup: broken links for CA certificates

### ISSUE THREAD

**Issue by wetneb** [kanthaus/kanthaus.online#44]
When going to:
https://kanthaus.online/en/blog/2019-08-27-kanthaus-wifi

The CA certificates link to:
https://kanthaus.online/en/blog/2019-08-27-kanthaus-wifi/kanthaus_ca.pem
https://kanthaus.online/en/blog/2019-08-27-kanthaus-wifi/kanthaus_ca.p12
both of which are 404 as of now. :)

**Comment by tiltec** [comment]
Files are in the repo, not sure why they can't be accessed. Maybe a grav update broke something? https://github.com/kanthaus/kanthaus.online/tree/master/user/pages/30.blog/2019-08-27-kanthaus-wifi

**Comment by DougInAMug** [comment]
Yeah... there is also now the handbook page: https://kanthaus.online/en/blog/2019-08-27-kanthaus-wifi

These seem to be duplicate info, one in EN, one in DE. I feel it makes more sense to have them in the handbook than the website.

@NerdyProjects @alangecker ?

**Comment by DougInAMug** [comment]
Aha! The kanthaus.online blog links _do_ work if you right-click and 'save link as...'

Seems like Grav expects a page? Strange...

**Comment by tiltec** [comment]
It might be related to MIME types... also the link on the handbook page doesn't work directly, you need to click "save link as": https://handbook.kanthaus.online/assets/wifi.html
